spec/models/ability_spec.rb

```diff
@@ -171,6 +171,35 @@
         end
       end
 
+      describe "just orga members, team\'s supervisor and team\'s students should be able to see offered conference for a team" do
+        let(:user) { FactoryGirl.create(:student)}
```

Comment:
Is it really necessary to `create` the user here? Wouldn't `build` be sufficient?